backend/src/v2/metadata/client.go

```diff
@@ -743,8 +746,8 @@ func (c *Client) GetArtifactName(ctx context.Context, artifactId int64) (string,
 	return getArtifactName(event.Path)
 }
 
-// GetArtifacts ...
-func (c *Client) GetArtifacts(ctx context.Context, ids []int64) ([]*pb.Artifact, error) {
+// GetArtifactsByID ...
```

Comment:
Can we remove this comment?

Reply:
This is to stay consistent with the rest of the codebase, I don't know how or why they standardized on this, but removing this standard should be a separate effort.